projects/gloo/cli/pkg/prerun/version_warning.go

```diff
@@ -0,0 +1,161 @@
+package prerun
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/options"
+	"github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/version"
+	version2 "github.com/solo-io/gloo/projects/gloo/pkg/api/grpc/version"
+	"github.com/solo-io/go-utils/errors"
+	"github.com/solo-io/go-utils/versionutils"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"strings"
+)
+
+func VersionMismatchWarning(opts *options.Options, cmd *cobra.Command) error {
+	return WarnOnMismatch(os.Args[0], version.NewKube(opts.Metadata.Namespace), &defaultLogger{})
```

Comment:
This will print the whole path to the executable, not sure if this is intentional. If we did 
```suggestion
	return WarnOnMismatch(path.Base(os.Args[0]), version.NewKube(opts.Metadata.Namespace), &defaultLogger{})
```

we would always just print the last part, i.e. `glooctl`, which I think is cleaner.

Reply:
Talked offline, `os.Args[0]` was intentional